src/mg/PAMI/Logger/Log4PhpAdapter.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace PAMI\Logger;
+
+use Logger;
+use LoggerLevel as Log4PhpLevel;
+use LoggerLevel;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel as PsrLevel;
+
+class Log4PhpAdapter implements LoggerInterface
+{
+
+    /**
+     * Contains the log4php logger.
+     *
+     * @var Logger
+     */
+    private $logger;
+
+    private static $logLevelAdapter = array(
+        PsrLevel::EMERGENCY => Log4PhpLevel::FATAL,
+        PsrLevel::ALERT     => Log4PhpLevel::FATAL,
+        PsrLevel::CRITICAL  => Log4PhpLevel::FATAL,
+        PsrLevel::ERROR     => Log4PhpLevel::ERROR,
+        PsrLevel::WARNING   => Log4PhpLevel::WARN,
+        PsrLevel::NOTICE    => Log4PhpLevel::INFO,
+        PsrLevel::INFO      => Log4PhpLevel::INFO,
+        PsrLevel::DEBUG     => Log4PhpLevel::DEBUG,
+    );
+
+    /**
+     * Creates a new instance of this class with a
+     * logger with the given name and configuration.
+     *
+     * @param string $name
+     * @param mixed  $config
+     */
+    public function __construct($name = 'main', $config = null)
+    {
+        Logger::configure($config);
+        $this->logger = Logger::getLogger($name);
+    }
+
+    public function emergency($message, array $context = array())
+    {
+        $this->log(PsrLevel::EMERGENCY, $message, $context);
+    }
+
+    public function alert($message, array $context = array())
+    {
+        $this->log(PsrLevel::ALERT, $message, $context);
+    }
+
+    public function critical($message, array $context = array())
+    {
+        $this->log(PsrLevel::CRITICAL, $message, $context);
+    }
+
+    public function error($message, array $context = array())
```

Comment:
Oops! Copy-paste error. I'll fix it in the next commit. Do  you prefer a rebase (keeps the history clean), or a new commit (which makes it easier to spot the changes)?